### ISSUE TITLE
Flexget install fails on OSMC 2020.03-1

### ISSUE THREAD

**Issue by bverc** [alombarte/raspberry-osmc-automated#39]
HDD died, so figured it was good time for a fresh reinstall. 

Installation script appears to work well on latest version of OSMC, however flexget never actually installs. 
>   File "/tmp/easy_install-M8VZkv/FlexGet-3.1.55/setup.py", line 2, in <module>
> ImportError: No module named pathlib
> Flexget will download content from RSS: http:\/\/showrss.info\/user\/[redacted].rss
> /home/osmc/.raspberry-osmc-automated/bash/install_flexget.sh: line 35: /usr/local/bin/flexget: No such file or directory

Thinking this would be as simple as installing pathlib first:
`sudo pip install pathlib`

However trying to install flexget again gives the new error:
>   File "/tmp/easy_install-T1FwtB/FlexGet-3.1.55/setup.py", line 6, in <module>
> AttributeError: 'PosixPath' object has no attribute 'read_text'

This looks like it is related to some sort of versioning conflict between python 2 and 3.

**Comment by bverc** [comment]
Ok so made some progress here. Able to get flexget installed by:
```
sudo apt-get install python3-pip
sudo pip3 install setuptools
sudo pip3 install wheel
sudo pip3 install pathlib
sudo easy_install flexget
```

However, running flexget fails:
> pkg_resources.DistributionNotFound: The 'immutables>=0.9' distribution was not found and is required by contextvars

Weirdly, immutables 0.14 installed in the previous step, so no clue why that is failing.

**Comment by bverc** [comment]
Further digging
"Starting from version 3.0.0, Flexget supports only Python 3.6 and higher"

So we can either install python 3.6 or downgrade flexget, and downgrading is the easiest option. Assuming the steps above have not been followed, simply run:
`sudo pip install -Iv flexget==2.21.35`

and voila!

**Comment by alombarte** [comment]
Hi @bverc , I haven't tested any of this for a looooong time. But if you feel like submitting a PR I will go through it. I am not proud of this quick and dirt scripting I did to accomlish the matter. I am guessing if I had to do it all over again I would create a docker image with fixed versions and libraries so we would be free of this python versions dependency madness. 

Thank you very much for sharing your findings!
